utils/swift_build_support/swift_build_support/products/indexstoredb.py

```diff
@@ -74,6 +74,10 @@ def get_dependencies(cls):
                 swiftpm.SwiftPM,
                 swiftsyntax.SwiftSyntax]
 
+    @classmethod
+    def has_cross_compile_hosts(self, args):
+        return args.cross_compile_hosts
```

Comment:
Maybe pull this into the base class so [all products can use it](https://github.com/apple/swift/blob/e9eb3b06c1e62d65062cdc44db7f33e673cf5345/utils/swift_build_support/swift_build_support/products/swiftdriver.py#L131)?

Reply:
Good idea :+1